spec/controllers/responses_controller_spec.rb

```diff
@@ -45,5 +45,10 @@
     it 'finishes the response' do
       expect { subject }.to change { Events::ResponseFinished.count }.by 1
     end
+
+    it 'sends the email' do
+      expect(ResponseMailer).to receive(:response_email).and_call_original
```

Comment:
Prima, als het logisch is om code op een andere manier aan te pakken door een wijziging in je PR vind ik het wel netjes om de rest van de codebase daar consistent mee te houden.